tools/processresults.py

```diff
@@ -229,19 +281,25 @@ def _WriteToCsv(results, canonical_index, out_file):
     csv_writer.writeheader()
```

Comment:
Adding this comment here because github doesn't let me comment on code that's not changed as part of this CL:

Looks like you could pull |field_names| calculation out of the |with| block and move it right after |canonical_questions| calculation at the top. That'll make this code a git easier to read.